circle.yml

```diff
@@ -0,0 +1,22 @@
+machine:
+  environment:
+    PATH: $HOME/.cabal/bin/:$PATH
+
+    # CircleCI supports up to 32 processors,
+    # but 8 processors yields the fastest results
+    PARALLEL_TEST_PROCESSORS: 8
+
+
+dependencies:
+
+  # Cache ShellCheck for subsequent builds
+  cache_directories:
+    - "~/.cabal"
```

Comment:
The tilde didn't work in `PATH`, and YML doesn't allow string interpolation for the `cache_directories` value, and I'm following the documented way there https://circleci.com/docs/configuration#cache-directories